modules/ldap-syncing-augmented-activedir.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-syncing-augmented-activedir_{context}"]
+= Syncing groups using the Augmented Active Directory schema
```

Comment:
Just a question on whether Augmented needs to be capitalized. Elsewhere it is not.